### ISSUE TITLE
Watchman prevents AppSync Simulator

### ISSUE THREAD

**Issue by adriantodt** [bboure/serverless-appsync-simulator#70]
**Describe the bug**
Trying to execute the AppSync Emulator briefly starts it, but crashes shortly after with a unhelpful error message. Searching it on Google doesn't seem to get anything else, really.

```
Serverless: Load command interactiveCli
[<omitted because there's a lot of commands>]
Serverless: Load command dev
Serverless: Invoke offline:start
AppSync Simulator: AppSync endpoint: http://192.168.15.107:20002/graphql
AppSync Simulator: GraphiQl: http://192.168.15.107:20002
Watchman:  spawn watchman ENOENT
 
  Error --------------------------------------------------
 
  Error: spawn watchman ENOENT
      at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
      at onErrorNT (internal/child_process.js:465:16)
      at processTicksAndRejections (internal/process/task_queues.js:80:21)
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     Operating System:          linux
     Node Version:              14.15.4
     Framework Version:         1.75.1
     Plugin Version:            3.6.16
     SDK Version:               2.3.1
     Components Version:        2.32.0
```

**To Reproduce**
My current `serverless.yml` file uses the following plugins

```
plugins:
  - serverless-appsync-plugin
  - serverless-pseudo-parameters
  - serverless-plugin-warmup
  - serverless-appsync-simulator
  - serverless-offline
  ```

**Expected behavior**
I just expected at least a better error message, honestly. I'm just trying to configure it to replace `@conduitvc/appsync-emulator-serverless`.

**Screenshots**
N/A

**Additional context**
N/A

**Comment by bboure** [comment]
Thanks for opening this @adriantodt 
Just to remove the obvious stuff out of the way, you have watchman installed globally, right?

What version are you using?

**Comment by cameroncf** [comment]
If you don't have watchman installed, you might also just try this to see if it will make the error to go away in the meantime...

custom:
  appsync-simulator:
    watch: false

**Comment by adriantodt** [comment]
Here's the yarn.lock snippet of the code:

```
serverless-appsync-simulator@^0.11.0:
  version "0.11.0"
  resolved "https://registry.yarnpkg.com/serverless-appsync-simulator/-/serverless-appsync-simulator-0.11.0.tgz#e753d9c147de034b1d596e30411d939c922beaa4"
  integrity sha512-LkU4DLwPfTXIVwn5hpnvJ9EF7ok7McT9a/NCOKFjGiXSnvboQomlb34mgolgBkfBdGveZmS23f0RLnf1kNg07A==
  dependencies:
    amplify-appsync-simulator "^1.23.9"
    amplify-nodejs-function-runtime-provider "^1.1.6"
    aws-sdk "^2.792.0"
    axios "^0.21.0"
    babel-jest "^26.6.3"
    cfn-resolver-lib "^1.1.7"
    dataloader "^2.0.0"
    fb-watchman "^2.0.1"
    jest "^26.6.3"
    lodash "^4.17.20"
    merge-graphql-schemas "^1.5.8"
```

Using `watch: false` does make the error go away (another error appears, though)

**Comment by bboure** [comment]
@adriantodt Did you solve this issue?
Do you have watchman installed globally? 
What other error do you see?

**Comment by adriantodt** [comment]
Oh sorry, my bad. My intent was to get the AppSync Simulator to load, so I did not read the README that thoroughly.
I had other errors related to serverless-offline which I managed to fix, eventually leading to #73.
I do not pretend to use the watch function, so I'll be closing this.

**Comment by Jasper8628** [comment]
thanks @cameroncf  for the answer, I had the same issue where serverless is stuck on "Building with Webpack...", had a quick look through all 31 closed issues, I decided to give this one a try haha
(I'm just adding more descriptions on the issue to make it more google-albe)
